docs/examples/config/image-registry/app-validate-imagePullSecret.yaml

```diff
@@ -1,14 +0,0 @@
-apiVersion: core.oam.dev/v1beta1
-kind: Application
```

Comment:
you need to write a new example of config

Reply:
Added